controllers/pwas/crud.js

```diff
@@ -16,17 +16,10 @@
 'use strict';
 
 const express = require('express');
-const config = require('../../config/config');
 const images = require('../../lib/images');
-const fetch = require('node-fetch');
-
-const PWA = 'PWA';
-
-function getModel() {
-  return require('../../lib/model-' + config.get('DATA_BACKEND'));
-}
-
+const pwaModel = require('../../models/pwa');
 const router = express.Router(); // eslint-disable-line new-cap
+const LIST_PAGE_SIZE = 10;
 
 // Set Content-Type for all responses for these routes
```

Comment:
suggestion: it might make sense to extract middlewares into their own package (https://www.terlici.com/2014/08/25/best-practices-express-structure.html).